### ISSUE TITLE
[BUG] Unexpected token '<', "<!DOCTYPE "... is not valid JSON

### ISSUE THREAD

**Issue by cdmathukiya** [rakutentech/laravel-request-docs#259]
**Describe the bug**
When I have define `Route::resource` in route file but not create all the methos in controller then it create this error.
![image](https://github.com/rakutentech/laravel-request-docs/assets/18529326/38407adf-25bf-4437-af4b-92b30eb00c17)

![image](https://github.com/rakutentech/laravel-request-docs/assets/18529326/f5df3599-375e-4942-8395-9331b21cd0fc)

**To Reproduce**
1. just define any route in route file.
2. do not create same method in controller

**Expected behavior**
Output should show all the available/valid endpoints.
![image](https://github.com/rakutentech/laravel-request-docs/assets/18529326/39073a3f-d902-4038-a493-e51bdda6a0af)

**Versions (please complete the following information):**
![image](https://github.com/rakutentech/laravel-request-docs/assets/18529326/a4316bf4-f19b-40cf-84b1-6fcbe5287e60)

**Comment by kevincobain2000** [comment]
Thanks. Will check this.

**Comment by quangtlu** [comment]
@cdmathukiya @kevincobain2000 Is there any way to fix this? I'm in a hurry, thank you

**Comment by kevincobain2000** [comment]
Is the debug true in the config of LRD?

**Comment by cdmathukiya** [comment]
No, it is false
![image](https://github.com/rakutentech/laravel-request-docs/assets/18529326/8c3ab1d2-175b-40b4-a61b-3f6137961f1b)

**Comment by kevincobain2000** [comment]
Thanks.
For the time being, you can try adding `->except(['show'])` to the resource for the functions that are not implemented...

**Comment by alpemreelmas** [comment]
> 

I can confirm that this worked for me. I had forgetten just "show" method to call inside except method but I got this error. After adding ->except(["update","edit"]) it worked.

**Comment by alpemreelmas** [comment]
> Thanks. For the time being, you can try adding `->except(['show'])` to the resource for the functions that are not implemented...

I can confirm that this worked for me. I had forgetten just "show" method to call inside except method but I got this error. After adding ->except(["update","edit"]) it worked.

**Comment by kevincobain2000** [comment]
All good then? Let me know if I can close this issue.

**Comment by alpemreelmas** [comment]
> 



> All good then? Let me know if I can close this issue.

Yes, you can close this issue
Thanks

**Comment by AshirNob** [comment]
HEY,
see attached having same issue. even if i comment my all  api routes, cleared cache this error doesn't go away.
also i removed my controller code still same.
what i am doing wrong here ?

![doct-type-isse](https://github.com/rakutentech/laravel-request-docs/assets/52866202/e7b08d85-2629-4bb5-8fd3-8dffc2cb308d)

**Comment by JanPomelo** [comment]
I have the same error as soon as I publish the assets.

**Comment by kevincobain2000** [comment]
You don't need to publish assets on v2

**Comment by JanPomelo** [comment]
Yes, but I want to add some stylings to the request page since the li and heading tags in the extra documentation are not visually different from the normal text. So I thought I publish it and track the changes in git too.

**Comment by kevincobain2000** [comment]
well...follow the traceback for hints. The error that you are seeing is when it is not able to connect to the API.

**Comment by JanPomelo** [comment]
Will do it. Thanks!

**Comment by vincentdev15** [comment]
Today the bug is still there.

**Comment by kevincobain2000** [comment]
it's not a bug, it's just that there is an error in your project somewhere.

**Comment by vincentdev15** [comment]
The package should work with no error without having to add ```->except()``` to the needed resource routes, no ?

Furthermore I have API resource routes and some additional routes and it generates this error because of the presence of the additional routes.

**Comment by kevincobain2000** [comment]
@vincentdev15 Kinda yes, you try it as you are able to reproduce.
I am afraid that if you put it under the try and catch, it's just show now routes and I ll be reading issues with title "[[BUG] Routes not showing"
At least right now it is better to see that these are not showing because of some error that has occurred.

I could be wrong, if you want to submit a PR, I'd be happy to merge it.

**Comment by alexgrint** [comment]
I have just the package routes, and the error is the same
![image](https://github.com/user-attachments/assets/60ecde20-99d0-494e-80b0-6e2e74a7be8f)
![image](https://github.com/user-attachments/assets/ec40d7f3-642a-4a82-8485-2a3f3a3d5dfa)

**Comment by kevincobain2000** [comment]
You can enable debug in the config.

**Comment by kevincobain2000** [comment]
There you go. Thanks for notifying @ERaufi
